### ISSUE TITLE
Only the first schema appears in index.ts file

### ISSUE THREAD

**Issue by yekver** [anymaniax/orval#286]
If there more than one configuration were defined and all/some of them are having the same `workspace` option, the resulted `index.ts` will be corrupted by [async writing operation](https://github.com/anymaniax/orval/blob/master/src/core/writers/schemas.ts#L90) and will contain the generated files only for the first provided configuration.

```javascript
// orval.config.js

const defaultOutputConfig = { workspace: 'src/api' }

export default defineConfig({
 api1: { input: '../api1.yaml', output: { ...defaultOutputConfig, target: './generated/api1.ts' } },
 api2: { input: '../api2yaml', output: { ...defaultOutputConfig, target: './generated/api2.ts' } },
 api3: { input: '../api3.yaml', output: { ...defaultOutputConfig, target: './generated/api3.ts' } }
```

Generated file misses the declarations for `api2.ts` and `api3.ts`
```javascript
// src/api/index.ts

export * from './generated/api1.ts`
```

**Comment by anymaniax** [comment]
will be fixed in the next version

**Comment by anymaniax** [comment]
fixed with `6.6.1`
